.github/workflows/main_workflow.yml

```diff
@@ -156,7 +156,6 @@ jobs:
           submodules: recursive
       - uses: actions/setup-python@master
         with:
-          submodules: recursive
```

Comment:
Is this not neccesary?

Reply:
No, this is probably a copy-paste from the checkout.

d5b38a5